cmd/go-filecoin/main.go

```diff
@@ -354,29 +355,29 @@ var priceOption = cmdkit.StringOption("gas-price", "Price (FIL e.g. 0.00013) to
 var limitOption = cmdkit.Uint64Option("gas-limit", "Maximum GasUnits this message is allowed to consume")
 var previewOption = cmdkit.BoolOption("preview", "Preview the Gas cost of this command without actually executing it")
 
-func parseGasOptions(req *cmds.Request) (types.AttoFIL, types.GasUnits, bool, error) {
+func parseGasOptions(req *cmds.Request) (types.AttoFIL, gas.Unit, bool, error) {
 	priceOption := req.Options["gas-price"]
 	if priceOption == nil {
-		return types.ZeroAttoFIL, types.GasUnits(0), false, errors.New("gas-price option is required")
+		return types.ZeroAttoFIL, gas.NewGas(0), false, errors.New("gas-price option is required")
```

Comment:
Nit: `gas.Unit(0)` is shorter and stutters less. We should delete `NewGas` entirely.

Reply:
Also for this case, `gas.Zero`.